sources/packages/api/src/route-controllers/note/note.aest.controller.ts

```diff
@@ -75,14 +75,13 @@ export class NotesController extends BaseController {
    * @param institutionId
    * @param payload Note create object.
    */
-  @Groups(UserGroups.AESTUser)
-  @AllowAuthorizedParty(AuthorizedParties.aest)
+
   @Roles(Role.InstitutionCreateNote)
   @Post("/institution/:institutionId")
   async addInstitutionNote(
     @UserToken() userToken: IUserToken,
     @Param("institutionId") institutionId: number,
-    @Body() payload: NoteBaseDTO,
+    @Body() payload: NoteBaseAPIInDTO,
   ): Promise<void> {
     const institutionNote = transformToNoteEntity(payload, userToken.userId);
     await this.institutionService.saveInstitutionNote(
```

Comment:
`saveInstitutionNote` is a method to be refactored to stop loading all the relations. Can you do it alongside this PR, please?

Reply:
Thank you for doing this non-PR related change 👍